src/components/books.jsx

```diff
@@ -1,24 +1,19 @@
-/* eslint-disable react/prop-types */
-import Form from './Form';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { Removebook } from '../redux/books/booksSlice';
 
-const Books = ({ title, author }) => (
-  <div>
-    <ul>
-      <div className="book flexRow">
-        {' '}
-        <li>
-          author:
-          {author}
-          {' '}
-          title:
-          {title}
-        </li>
-        <button type="submit" className="remove">remove</button>
+const Book = ({ title, author, id }) => {
+  const dispatch = useDispatch();
+  return (
+    <div className="book flexRow">
+      <h2>{author}</h2>
+      {' '}
+      <p>{title}</p>
+      <div className="actions">
+        <button type="button" onClick={() => dispatch(Removebook(id))}>remove</button>
       </div>
-    </ul>
-    <Form />
-  </div>
+    </div>
```

Comment:
- [ ] something seems to be going wrong after removing a book, because after a user removes a book and they go to the Category page one book is still visible as you can see in the below example:
![afbeelding](https://user-images.githubusercontent.com/39240820/229834605-9a27ac66-45cf-4bf6-83fb-d1338fe11624.png)
Books aren't supposed to be visible on the category page, please fix this before requesting another review.

Reply:
Am suggesting you have a look at this short video showing you what I have on my side because it's working quite well.

loom video: https://www.loom.com/share/2953c4a7d85d4382beec53f1e57028f2